### PR TITLE
Add Android scan sample, make embedded iOS sample conditional and bump NativeScript version

### PR DESCRIPTION
**Pull request by steinerj**
After much confusion in the issues, I have added an Android scanning sample which should work on all API levels of Android.
The iOS custom view element is conditional and will not trigger an error on Android. Thus, this project should safely work on all platforms.

### CODE REVIEW

File: demo-ng/app/item/item-detail.component.ts [hunk 1]
```diff
@@ -1,28 +1,63 @@
-import {Component, OnInit} from "@angular/core";
-import {ActivatedRoute} from "@angular/router";
+import { ScanOptions } from './../../platforms/android/app/src/main/assets/app/tns_modules/nativescript-barcodescanner/barcodescanner-common.d';
```
Comment (EddyVerbruggen) [inline demo-ng/app/item/item-detail.component.ts]: I'm afraid your IDE resolved the wrong import, can you fix it perhaps?

`import { BarcodeScanner, ScanOptions } from "nativescript-barcodescanner";`

**Comment by EddyVerbruggen** [comment]
Great work!! Thanks for taking the time making this demo less confusing 🤪

**Comment by r0zar** [comment]
Ty Eddy and steiner for the library support!

File: demo-ng/app/item/item-detail.component.ts (hunk 1 above)
Comment (steinerj) [reply demo-ng/app/item/item-detail.component.ts]: Oops. That's pushing on Friday evenings is a "no-no"... even for minor sample stuff. 
Cheers for catching it, @EddyVerbruggen and great plugin!

**Pull request merged**